01research/sec03CMake.md

```diff
@@ -63,11 +63,12 @@ to google CMake issues and work with CMake.
 Demo an "out-of-source" build
 
 ``` bash
-cd ~/build
+mkdir cmake_demo
+cd cmake_demo
 git clone https://github.com/MattClarkson/CMakeHelloWorld
 mkdir CMakeHelloWorld-build
 cd CMakeHelloWorld-build
-ccmake ../CMakeHelloWorld
+cmake ../CMakeHelloWorld
```

Comment:
I normally prefer to have the `build` directory under the `src`. It makes my directories to look a bit neater. But that's me ;)

Reply:
Agreed, I've pushed the change so build is a subdirectory